app/routes/events/components/EventEditor/index.js

```diff
@@ -637,6 +637,10 @@ const validate = (data) => {
   if (!moment(data.startTime).isBefore(data.endTime)) {
     errors.endTime = 'Starttidspunkt må være før sluttidspunkt';
   }
+  if (!moment(data.unregistrationDeadline).add(1, 'days').isSameOrBefore(moment(data.paymentDueDate))) {
```

Comment:
What's with the negation on these checks? Is it not more readable to use `isSameOrAfter`? In this case just `isAfter`, I suppose.

Reply:
Not sure about why the first one is negated, but the second one I'll fix, nice spotted! I'll just change the first one as well while I'm at it.